html/components/masthead.js

```diff
@@ -116,7 +116,7 @@ const showSelectorMask = mastheadSelectorMask => {
 
 const bindUIEvents = () => {
   getElements('[data-sprk-mobile-nav-trigger]', element => {
-    const mainLayout = document.querySelector('[data-sprk-main]');
```

Comment:
with this update, what happens to the [data-sprk-main] selector? does the masthead no longer need it?

Reply:
Yeah, good catch. We don't use that attribute for anything else in the masthead. I've updated the story, tests, PR to reflect it.